_licenses/odbl-1.0.txt

```diff
@@ -0,0 +1,534 @@
+---
+title: ODC Open Database License v1.0
+spdx-id:  ODbL-1.0
+nickname: ODbL
+
+description: The Open Database License (ODbL) is a license agreement intended to allow users to freely share, modify, and use a database while maintaining this same freedom for others.
+
+how: Create a text file (typically named LICENSE or LICENSE.txt) in the root of your source code and copy the text of the license into the file.
+
+using:
+  - Open Food Facts: https://openfoodfacts.org/
+  - OpenCorporates: https://opencorporates.com/
+  - OpenStreetMap: https://openstreetmap.org/
+  - RATP (Paris region transit authority): https://data.ratp.fr/
```

Comment:
@johanricher there should be 3 examples, and the links should be to license files that can be matched against the license text added. See implementation for other licenses:

```
$ grep ': http' *.txt
0bsd.txt:  - PickMeUp: https://github.com/nazar-pc/PickMeUp/blob/master/copying.md
0bsd.txt:  - smoltcp: https://github.com/m-labs/smoltcp/blob/master/LICENSE-0BSD.txt
0bsd.txt:  - Toybox: https://github.com/landley/toybox/blob/master/LICENSE
apache-2.0.txt:  - Kubernetes: https://github.com/kubernetes/kubernetes/blob/master/LICENSE
apache-2.0.txt:  - PDF.js: https://github.com/mozilla/pdf.js/blob/master/LICENSE
apache-2.0.txt:  - Swift: https://github.com/apple/swift/blob/master/LICENSE.txt
bsd-2-clause.txt:  - go-redis: https://github.com/go-redis/redis/blob/master/LICENSE
bsd-2-clause.txt:  - Homebrew: https://github.com/Homebrew/brew/blob/master/LICENSE.txt
bsd-2-clause.txt:  - Pony: https://github.com/ponylang/ponyc/blob/master/LICENSE
bsd-3-clause.txt:  - d3: https://github.com/d3/d3/blob/master/LICENSE
bsd-3-clause.txt:  - LevelDB: https://github.com/google/leveldb/blob/master/LICENSE
bsd-3-clause.txt:  - Quill: https://github.com/quilljs/quill/blob/develop/LICENSE
ecl-2.0.txt:  - Sakai: https://github.com/sakaiproject/sakai/blob/master/LICENSE
ecl-2.0.txt:  - OAE: https://github.com/oaeproject/Hilary/blob/master/LICENSE
ecl-2.0.txt:  - Opencast: https://github.com/opencast/opencast/blob/develop/LICENSE
epl-1.0.txt:  - Eclipse hawkBit: https://github.com/eclipse/hawkbit/blob/master/LICENSE
epl-1.0.txt:  - JUnit: https://github.com/junit-team/junit4/blob/master/LICENSE-junit.txt
epl-1.0.txt:  - openHAB: https://github.com/openhab/openhab-distro/blob/master/LICENSE
epl-2.0.txt:  - Eclipse Ditto: https://github.com/eclipse/ditto/blob/master/LICENSE
epl-2.0.txt:  - Eclipse SmartHome: https://github.com/eclipse/smarthome/blob/master/LICENSE
epl-2.0.txt:  - SUMO: https://github.com/eclipse/sumo/blob/master/LICENSE
eupl-1.2.txt:  - AethysRotation: https://github.com/SimCMinMax/AethysRotation/blob/master/LICENSE
eupl-1.2.txt:  - WildDuck: https://github.com/nodemailer/wildduck/blob/master/LICENSE
eupl-1.2.txt:  - ZoneMTA: https://github.com/zone-eu/zone-mta/blob/master/LICENSE
gpl-2.0.txt:  - AliSQL: https://github.com/alibaba/AliSQL/blob/master/COPYING
gpl-2.0.txt:  - Discourse: https://github.com/discourse/discourse/blob/master/LICENSE.txt
gpl-2.0.txt:  - Joomla!: https://github.com/joomla/joomla-cms/blob/staging/LICENSE.txt
gpl-3.0.txt:  - Ansible: https://github.com/ansible/ansible/blob/devel/COPYING
gpl-3.0.txt:  - Bash: https://git.savannah.gnu.org/cgit/bash.git/tree/COPYING
gpl-3.0.txt:  - GIMP: https://git.gnome.org/browse/gimp/tree/COPYING
isc.txt:  - Helix: https://github.com/tildeio/helix/blob/master/LICENSE
isc.txt:  - Node.js semver: https://github.com/npm/node-semver/blob/master/LICENSE
isc.txt:  - OpenStreetMap iD: https://github.com/openstreetmap/iD/blob/master/LICENSE.md
mit.txt:  - Babel: https://github.com/babel/babel/blob/master/LICENSE
mit.txt:  - .NET Core: https://github.com/dotnet/corefx/blob/master/LICENSE.TXT
mit.txt:  - Rails: https://github.com/rails/rails/blob/master/MIT-LICENSE
mpl-2.0.txt:  - Servo: https://github.com/servo/servo/blob/master/LICENSE
mpl-2.0.txt:  - Syncthing: https://github.com/syncthing/syncthing/blob/master/LICENSE
mpl-2.0.txt:  - TimelineJS3: https://github.com/NUKnightLab/TimelineJS3/blob/master/LICENSE
ncsa.txt: - LLDB: https://github.com/llvm-mirror/lldb/blob/master/LICENSE.TXT
ncsa.txt: - ROCR-Runtime: https://github.com/RadeonOpenCompute/ROCR-Runtime/blob/master/LICENSE.txt
ncsa.txt: - RLTK: https://github.com/chriswailes/RLTK/blob/master/LICENSE
odbl-1.0.txt:source: http://opendatacommons.org/licenses/odbl/1.0/
odbl-1.0.txt:  - OpenStreetMap: https://wiki.openstreetmap.org/wiki/Open_Database_License
odbl-1.0.txt:  - ProtonDB: https://github.com/bdefore/protondb-data
odbl-1.0.txt:  - Open Food Facts: https://world.openfoodfacts.org/data
osl-3.0.txt:  - appserver.io: https://github.com/appserver-io/appserver/blob/master/LICENSE.txt
osl-3.0.txt:  - JsonMapper: https://github.com/cweiske/jsonmapper/blob/master/LICENSE
osl-3.0.txt:  - Restyaboard: https://github.com/RestyaPlatform/board/blob/master/LICENSE.txt
postgresql.txt:  - pgBadger: https://github.com/darold/pgbadger/blob/master/LICENSE
postgresql.txt:  - pgAdmin: https://github.com/postgres/pgadmin4/blob/master/LICENSE
postgresql.txt:  - .NET Access to PostgreSQL: https://github.com/npgsql/npgsql/blob/dev/LICENSE
unlicense.txt:  - youtube-dl: https://github.com/rg3/youtube-dl/blob/master/LICENSE
unlicense.txt:  - kakoune: https://github.com/mawww/kakoune/blob/master/UNLICENSE
unlicense.txt:  - RDF.rb: https://github.com/ruby-rdf/rdf/blob/master/UNLICENSE
upl-1.0.txt:  - WebLogic Kubernetes Operator: https://github.com/oracle/weblogic-kubernetes-operator/blob/master/LICENSE.txt
upl-1.0.txt:  - Oracle Product Images for Docker: https://github.com/oracle/docker-images/blob/master/LICENSE
upl-1.0.txt:  - Oracle Product Boxes for Vagrant: https://github.com/oracle/vagrant-boxes/blob/master/LICENSE
```

Reply:
Good catch. It should be good now.

Reply:
The examples should point to text files (or HTML renderings of text files in the case file is in a github.com repo). Take a closer look at the examples from other licenses above.

Reply:
Those examples are licence files for code stored on a Github repository. ODbL being a licence applied to databases, I can't do the same.
Creative Commons, Open Font License, EUPL as well as others were accepted without any examples given...

Reply:
Those licenses were added before the requirement and tests were, but are used plenty in git repos. If there aren't examples of such use of ODbL then it isn't appropriate to add it to choosealicense.com; producing such examples is intended to be part of [vetting whether adding a license is appropriate](https://github.com/github/choosealicense.com/blob/gh-pages/CONTRIBUTING.md#adding-a-license).

Here's a search on this site that might turn up plausible examples
https://github.com/search?q=filename%3ALICENSE+%22Derivative+Databases+and+Produced+Works%22

Reply:
[As I've noted before](https://github.com/github/choosealicense.com/pull/195#issuecomment-453743367) ODbL passes the Github test since it's used on [more than 2000 github repos](https://github.com/search?q=ODbL+filename%3ALICENSE). But it wasn't my interpretation that the "3 notable projects" should also be from Github. i'll try to find three that are notable enough.

Reply:
Here are some Github projects that I've found, sorted by number of stars:

* https://github.com/mledoze/countries/
* https://github.com/svaksha/Julia.jl/
* https://github.com/jpatokal/openflights ([pending this PR](https://github.com/jpatokal/openflights/pull/950))
* https://github.com/zonedb/zonedb/
* https://github.com/wiredmax/world-currencies
* https://github.com/osmlab/appledata (although the file are not stored on the repo)
* https://github.com/healthsites/healthsites
* https://github.com/paperscape/paperscape-data
* https://github.com/opennorth/represent-canada-data

I'll update it if come up with better projects. If not, would that the top 3 be good enough?

Reply:
Some of them probably would work. They need to have a license file that matches the text being added in this PR. I assume your openflights PR does that for that repo. At a glance https://github.com/zonedb/zonedb/blob/master/LICENSE.md probably will pass the test which checks for matches. The first two don't include the whole license text, or add to it, so aren't likely to match.

Reply:
Ok so I've managed to update some of the repos I listed above, in particular #1 and #3 which are quite notable.
#2 already has ODbL as one of its licences, [in a dedicated file](https://github.com/svaksha/Julia.jl/blob/master/LICENSE-ODbL-1-0.md), with a copyright notice added at the beginning. [The maintainer wants to keep it](https://github.com/svaksha/Julia.jl/pull/156#pullrequestreview-231420429) and I suppose he has the right to.
What do you think @mlinksva? Could we find a way for the repo to pass the test while keeping the copyright notice? What are the good practices for copyright notices, should they be in a separate file?

Reply:
There isn't a way to pass the test while keeping the copyright notice. Ideally it would be moved to the README and/or a NOTICE file and/or file headers in source files. However of course any particular maintainer can do whatever they want. It isn't a big deal, just find another example.

Reply:
Alright. :)

I'll replace the links by:
- World Countries: https://github.com/mledoze/countries/blob/master/LICENSE
- OpenFlights: https://github.com/jpatokal/openflights/blob/master/data/LICENSE
- Public Zone Database: https://github.com/zonedb/zonedb/blob/master/LICENSE.md

If you confirm it's A-okay, I'll push the final commit.

Reply:
👍

Reply:
Done!